src/ssl.c

```diff
@@ -21497,11 +21497,23 @@ static int wolfSSL_DupSessionEx(const WOLFSSL_SESSION* input,
                 ret = WOLFSSL_FAILURE;
             }
             else {
+#ifndef XREALLOC
+                tmp = (byte*)XMALLOC(input->ticketLen,
+                        output->heap, DYNAMIC_TYPE_SESSION_TICK);
+                if (tmp != NULL)
+                    XMEMCPY(tmp, ticBuff, input->ticketLen);
```

Comment:
Should this copy ticLenAlloc instead of input->ticketLen to try and avoid reading over the end of 'ticBuff'?

Reply:
Good catch! Updated and also fixed a build issue with `WOLFSSL_NO_REALLOC` defined.

Reply:
yeah, good catch I missed it during my review.  Is this copy really necessary then? (is it overwritten by the memcpy on 21529? )

Reply:
Hmmm. Interesting. Yeah, I hadn't even considered that. I love removing memcpy's